src/tensorwaves/model.py

```diff
@@ -180,3 +183,49 @@ def variables(self) -> FrozenSet[str]:
     @property
     def argument_order(self) -> Tuple[str, ...]:
         return tuple(x.name for x in self.__argument_order)
+
+
+def add_components(
```

Comment:
How about we put this into some new module, since this is quite special code related to the `HelicityModel`.

Reply:
Yes, I suppose we need to re-introduce the `physics` module (but now separate from the `model` module). #15, #14, and #38 could be put there as well.

Reply:
#15 and #38 belong together (model domain related)

#14 not related to these, but related to this PR. So we could def put those together. Maybe into `fit_fractions.py` or `components.py`.

I simply do not want `model.py` to contain model specific implementations.

Reply:
Ok, moved them to a separate (possibly temporary) `physics` module (b2a190d). The location of these functions will become clearer in through those other issues